### PR TITLE
another batch of minor style fixes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86, @migueldiascosta please review?

File: easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-CrayGNU-2015.11.eb [hunk 1]
```diff
@@ -14,7 +14,7 @@ source_urls = ['https://www-user.tu-chemnitz.de/~potts/nfft/download/']
 sources = [SOURCELOWER_TAR_GZ]
 
 dependencies = [
-     ('fftw/3.3.4.5', EXTERNAL_MODULE),
+    ('fftw/3.3.4.3', EXTERNAL_MODULE),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-CrayGNU-2015.11.eb]: not just a style fix, was the change in version number intended?

Comment (boegel) [inline easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-CrayGNU-2015.11.eb]: oh, nice catch, copy-paste error, will fix

File: easybuild/easyconfigs/o/OBITools/OBITools-1.2.9-foss-2016a-Python-2.7.11.eb [hunk 2]
```diff
@@ -5,17 +5,15 @@ version = '1.2.9'
 versionsuffix = '-Python-%(pyver)s'
 
 homepage = 'http://www.grenoble.prabi.fr/trac/OBITools'
-description = """ The OBITools programs aims to help you to manipulate various data and sequence files in a convenient way using
- the Unix command line interface. They follow the standard Unix interface for command line program, allowing to chain a set of
- commands using the pipe mecanism. 
-"""
+description = """The OBITools programs aims to help you to manipulate various data and sequence files in a
+ convenient way using the Unix command line interface. They follow the standard Unix interface for command line
+ program, allowing to chain a set of commands using the pipe mecanism. """
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OBITools/OBITools-1.2.9-foss-2016a-Python-2.7.11.eb]: delete whitespace at end of last sentence?

File: easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb [hunk 3]
```diff
@@ -27,7 +27,8 @@ builddependencies = [
     ('Eigen', '3.2.7'),
 ]
 
-configopts = '-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN -DCMAKE_CXX_FLAGS="$LIBLAPACK_MT -DEIGEN_USE_MKL_ALL"'
+configopts = "-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN "
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb]: single quotes?

File: easybuild/easyconfigs/r/RAxML/RAxML-7.7.5-ictce-5.3.0-mt-sse3.eb [hunk 4]
```diff
@@ -15,7 +15,8 @@ source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
 
 skipsteps = ['configure', 'install']
 
-buildopts = '-f Makefile.SSE3.PTHREADS.gcc CC="$CC" && mkdir -p %(installdir)s/bin && cp raxmlHPC-PTHREADS-SSE3 %(installdir)s/bin'
+buildopts = '-f Makefile.SSE3.PTHREADS.gcc CC="$CC" && '
+buildopts += "mkdir -p %(installdir)s/bin && cp raxmlHPC-PTHREADS-SSE3 %(installdir)s/bin"
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/RAxML/RAxML-7.7.5-ictce-5.3.0-mt-sse3.eb]: single<->double quotes on the last lines?

File: easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb]: I tend to use double quotes for strings that include spaces, unless there's a technical reason not to (e.g. double quotes required in the string itself).

See WIP version of easyconfigs style guide at http://boegel-eb.readthedocs.io/en/easyconfigs_style/Code_style.html#code-style-easyconfigs-string-quotes

File: easybuild/easyconfigs/r/RAxML/RAxML-7.7.5-ictce-5.3.0-mt-sse3.eb (hunk 4 above)
Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-7.7.5-ictce-5.3.0-mt-sse3.eb]: same here: contains spaces, hence `"`

line above requires use of `"..."` to ensure bash expansion of `$CC`, so there I use single quotes for the string

**Comment by boegel** [comment]
Thanks for the thorough review @wpoely86 and @migueldiascosta!

**Pull request merged**